src/InvokePlaster.ps1

```diff
@@ -1,4 +1,4 @@
-<#
+﻿<#
```

Comment:
Ah. Can you replace those characters with this:
```
###############################################################################
# Helper functions
###############################################################################
```
And adjust the encoding.  Then I think we'll be ready to accept this PR.  Thanks!